api/ruby/find-inactive-members/find_inactive_members.rb

```diff
@@ -278,6 +367,12 @@ def member_activity
 Octokit.configure do |kit|
   kit.auto_paginate = true
   kit.middleware = stack if @debug
+  kit.connection_options = {
```

Comment:
Curious why you added this configuration?

Reply:
Hi @spier, thank you for taking your time to review my PR. IIRC, the default values for both request.open_timeout and timeout are 60s. I adjusted the value since it was taking a lot of time for me to retry when it failed.
I used this script in conjunction with bash script (I forget the exact script):
```
while true; do ruby find_inactive_members.rb; sleep 2; done
```